queries/perl/highlights.scm

```diff
@@ -29,6 +29,9 @@
   "foreach"
 ] @keyword.repeat
 
+("continue" @keyword.repeat
```

Comment:
Can't it just be `"continue" @keyword.repeat`

Reply:
I understand why now. https://perldoc.perl.org/functions/continue#continue-BLOCK